pkg/cmd/cli/cmd/process.go

```diff
@@ -58,5 +100,34 @@ Examples:
 		},
 	}
 	cmd.Flags().StringP("filename", "f", "", "Filename or URL to file to use to update the resource")
+	cmd.Flags().StringP("value", "v", "", "Set value for the specified parameter")
```

Comment:
We're doing exactly that in STI (see [here](https://github.com/openshift/source-to-image/blob/master/cmd/sti/main.go#L84)) so I agree with @smarterclayton with that version.